checkbox-provider/snap/hooks/install-deps

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I wonder if this (and `remove-deps`) should be placed somewhere else since they are not technically hooks into snapd events. I see a `bin/install-full-deps` file - could you perhaps replace that with this one? And add `remove-deps` to that same directory?

Reply:
let me see if a the install hook can access the scripts  in bin folder at this stage

Reply:
for the `remove-deps`, i don't think that is necessary:
- for automated testing, the system will be erased,  
- for manual testing, users might already have their setup and qat packages installed and they do not wish the provider to remove them,

Reply:
Fixed for using `bin/install-full-deps`

Reply:
> for the remove-deps, i don't think that is necessary:
> * for automated testing, the system will be erased,
> * for manual testing, users might already have their setup and qat packages installed and they do not wish the provider to remove them,

Can we just remove the `remove-deps` script altogether in that case?

If you want to keep it for convenience, I think it should be moved to a different folder as it is not a hook. The `snap/hooks` directory is a special directory for hooks with very specific naming requirements depending on the type of hook, see: https://ubuntu.com/robotics/docs/snap-configurations-and-hooks

Reply:
Thanks, makes sens